apps/recovery-portal/src/main/webapp/password-reset-complete.jsp

```diff
@@ -190,27 +192,56 @@
     <jsp:include page="includes/header.jsp"/>
     <% } %>
 </head>
-<body>
-    <div>
-        <form id="callbackForm" name="callbackForm" method="post" action="/commonauth">
-            <div>
-                <input type="hidden" name="username" value="<%=Encode.forHtmlAttribute(username)%>"/>
-            </div>
-            <div>
-                <input type="hidden" name="sessionDataKey" value="<%=Encode.forHtmlAttribute(sessionDataKey)%>"/>
-            </div>
-        </form>
-    </div>
+<body class="login-portal layout">
+    <% if (!RECOVERY_TYPE_INVITE.equalsIgnoreCase(type)) { %>
```

Comment:
If `type=invite` is not there in the password set link, what would be the behavior? Do we provide the previous behavior then?

Reply:
yes, it would follow the previous behaviour.